pkg/watermark/processor/processor_manager.go

```diff
@@ -244,18 +242,6 @@ func (v *ProcessorManager) startTimeLineWatcher() {
 			switch value.Operation() {
 			case store.KVPut:
 				p := v.GetProcessor(value.Key())
```

Comment:
```suggestion
				// a new processor's OT might take up to 5 secs to be reflected because we are not waiting for it to be added. 
				// This should not be a problem because the processor will send heartbeat as soon as it boots up.
				// In case we miss it, we might see a delay.
				p := v.GetProcessor(value.Key())
```